### ISSUE TITLE
TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.

### ISSUE THREAD

**Issue by woshibt** [wbkd/react-flow#2316]
### Describe the Bug

We are using `react-flow-render` to render graphs in a couple of pages including those viewing and editing the graphs. We upgraded `react-flow-render` to the latest version yesterday and made everything work except from time to time when switch in and out to a page containing the `react-flow-render` component the error shown in the title got popped up. 

A quick search showed it comes from the line: https://github.com/wbkd/react-flow/blob/7a909e942527daecb01fb78a4cfc00d5f46330ab/src/store/index.ts#L48

### Steps to Reproduce the Bug or Issue

It is not a published website. So we don't have a good repro to show here.  But below are the code sitting in the core functionality of our component using `react-flow-renderer`:

```
...
const dagreGraph = new dagre.graphlib.Graph()
dagreGraph.setDefaultEdgeLabel(() => ({}))

const nodeTypes = {
  defaultNode: DefaultNode,
  indexedNode: IndexedNode,
  selectedNode: SelectedNode,
}

const nodeWidth = 172
const nodeHeight = 36
const getRenderedNodes = (newNodes, newEdges, direction = 'LR') => {
  const isHorizontal = direction === 'LR'
  dagreGraph.setGraph({ rankdir: direction })

  newNodes.forEach((node) => {
    dagreGraph.setNode(node.id, { width: nodeWidth, height: nodeHeight })
  })

  newEdges.forEach((edge) => {
    dagreGraph.setEdge(edge.source, edge.target)
  })


  dagre.layout(dagreGraph)
  newNodes.forEach((node) => {
    const nodeWithPosition = dagreGraph.node(node.id)
    node.targetPosition = isHorizontal ? 'left' : 'top'
    node.sourcePosition = isHorizontal ? 'right' : 'bottom'

    // We are shifting the dagre node position (anchor=center center) to the top left
    // so it matches the React Flow node anchor point (top left).
    node.position = {
      x: nodeWithPosition.x - nodeWidth / 2,
      y: nodeWithPosition.y - nodeHeight / 2,
    }

    return node
  })

  return newNodes
}

const initRender = (nodes, edges, selected) => {
  const initNodes = nodes.map((e) => {
    // Add 'isSelected' to the node data, so that the nodes can be rendered accordingly later.
    let el = {
      ...e,
      data: {
        ...e.data,
        isSelected: selected.map((concept) => concept._key).includes(e.id),
      },
    }

    return {
      ...el,
      position: { x: 0, y: 0 },
      type: el.data.isSelected
        ? 'selectedNode'
        : e.indexed
          ? 'indexedNode'
          : 'defaultNode',
    }
  })

  const initEdges = edges.map((e) => {
    return { ...e, labelShowBg: false, arrowHeadType: 'arrowclosed' }
  })

  const renderedNodes = getRenderedNodes(initNodes, initEdges)
  return { renderedNodes, renderedEdges: initEdges }
}

...
export default function graphEditor({
  initialNodes,
  initialEdges,
  selected = [],
  onSelect,
}) {
  const reactFlowInstance = useReactFlow();
  const { renderedNodes, renderedEdges } = initRender(initialNodes, initialEdges, selected)
  const [nodes, setNodes, onNodesChange] = useNodesState(renderedNodes)
  const [edges, setEdges, onEdgesChange] = useEdgesState(renderedEdges)
...
      <ReactFlow
        nodes={nodes}
        edges={edges}
        onNodesChange={onNodesChange}
        onEdgesChange={onEdgesChange}
        nodeTypes={nodeTypes}
        onNodeContextMenu={onNodeContextMenu}
      >
        <Controls />
      </ReactFlow>
...
```

### Expected behavior

No errors shown

### Screenshots or Videos

<img width="945" alt="image" src="https://user-images.githubusercontent.com/5379948/180660300-301533e5-321c-4475-b39e-d2b4fd9a450e.png">

### Platform

- OS: macOS
- Browser: Chrome
- Version: 10.3.11

**Comment by moklick** [comment]
Thanks! We will check it.

**Comment by moklick** [comment]
Could you try if the issue still occurs when you use the `next` release: `npm install react-flow-renderer@next`

**Comment by woshibt** [comment]
The error seems fixed with v10.3.12.  Thank you!
